src/TabPane.js

```diff
@@ -155,21 +154,19 @@ class TabPane extends React.Component {
     return tabContent && tabContent.animation;
   }
 
-  handleEnter() {
-    const tabContent = this.context.$bs_tabContent;
-    if (!tabContent) {
-      return;
-    }
+  handleEnter(...args) {
+    if (this.props.onEnter) this.props.onEnter(...args);
```

Comment:
we should call `props.onEnter` after running our own `onEnter` handler, for consistency with elsewhere, no?

Reply:
probably, tho this was how it was before so idk

Reply:
the `createChainedFunction` put `props.onEnter` after `this.handleEnter`, though.

maybe just switch up the branch below to `if (tabContent)`?